src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,51 @@
+@use "../utilis/variables" as *; 
+
+.calendar {
+  display: flex;
+  justify-content: flex-start;
+  width: $width-calendar;
+  flex-wrap: wrap;
+
+  &__day {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+    min-width: $width-day;
+    height: $height-day;
+    background-color: $background-color-day;
+    border: 1px solid black;
+
+    margin-top: $margin-day;
+    margin-right: $margin-day;
+
+    cursor: pointer;
+
+    &:hover {
+      background-color: pink;
+      transform: translate(0, -20px);
+      transition: all 0.5s;
```

Comment:
move `transition` out from `:hover` (in this case, it'll work in 2 ways)
list properties you need instead of `all` (better for performance)